website_event_form/templates/snippets.xml

```diff
@@ -0,0 +1,122 @@
+<odoo>
+    <!-- New Events Form -->
+  <template id="s_event_form" name="Events Form">
+    <section class="oe_snippet_body oe_event_form">
+        <div class="container">
+            <div class="jumbotron mt-3 mb-3">
+                <h2 class="text-center text-primary">Add An Event</h2>
+            </div>
+            <div id="event_process_form">
+                <div class="row">
+                    <div class="col-md-6 form-group mb-2">
+                        <label for="event_user_name">Name</label>
+                        <input type="text" class="form-control"
+                               id="event_user_name"
+                               required="required"
+                               name="event_user_name"
+                               aria-describedby="nameHelp"
+                               placeholder="Enter your name"/>
+                        <small id="nameHelp" class="form-text text-muted"></small>
+                    </div>
+                    <div class="col-md-6 form-group mb-2">
+                        <label for="event_email_address">Email address</label>
+                        <input type="email" class="form-control"
+                               name="event_email"
+                               id="event_email_address"
+                               aria-describedby="emailHelp"
+                               placeholder="Enter email"/>
+                        <small id="emailHelp" class="form-text text-muted"></small>
+                    </div>
+                    <div class="col-md-6 form-group mb-2">
+                        <label for="event_name">Event</label>
+                        <input type="text" class="form-control"
+                               id="event_name"
+                               name="name"
+                               required="required"
+                               aria-describedby="eventHelp"
+                               placeholder="Enter Event Name"/>
+                        <small id="eventHelp" class="form-text text-muted"></small>
+                    </div>
+                    <div class="col-md-6 form-group mb-2">
+                        <label for="event_phone_no">Phone No.</label>
+                        <input type="text" class="form-control"
+                               required="required"
+                               name="event_phone_no"
+                               id="event_phone_no" placeholder="Phone No."/>
+                    </div>
+                    <div class="col-md-6 form-group mb-2">
+                        <label for="event_datetime">Start Date</label>
+                        <input type="text"
+                               id="event_start_date"
+                               name="date_begin"
+                               required="required"
+                               placeholder="2024-01-26 02:39:40"
```

Comment:
The place holder should be Insert Date - Mandatory   : format YYYY-MM-DD HH:MM:SS
Even better.... to avoid formatting errors use type date and odoo widget date.